CLEO5.vcxproj

```diff
@@ -160,29 +160,30 @@
       <FunctionLevelLinking>true</FunctionLevelLinking>
       <IntrinsicFunctions>true</IntrinsicFunctions>
       <SDLCheck>true</SDLCheck>
-      <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
+      <RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>
```

Comment:
not a fan of this. we need to include runtime dll to ZIP archive as CLEO installation should be simply unpacking.

what's the difference in VT report without this change?

Reply:
Restored static linking of runtime